checker/raw_result.go

```diff
@@ -286,11 +287,11 @@ type ArchivedStatus struct {
 // File represents a file.
 type File struct {
 	Path      string
-	Snippet   string   // Snippet of code
-	Offset    uint     // Offset in the file of Path (line for source/text files).
-	EndOffset uint     // End of offset in the file, e.g. if the command spans multiple lines.
-	FileSize  uint     // Total size of file.
-	Type      FileType // Type of file.
+	Snippet   string           // Snippet of code
+	Offset    uint             // Offset in the file of Path (line for source/text files).
+	EndOffset uint             // End of offset in the file, e.g. if the command spans multiple lines.
+	FileSize  uint             // Total size of file.
+	Type      finding.FileType // Type of file.
 	// TODO: add hash.
 }
```

Comment:
agreed. Since the raw results are not public yet, we could merge the two structure and use the same. Filed https://github.com/ossf/scorecard/issues/2626